### PR TITLE
#175: improvement for import

### PR DESCRIPTION
**Pull request by hohwille**
Improvement for #175 and #353

### CODE REVIEW

File: scripts/src/main/resources/scripts/command/ide [hunk 1]
```diff
@@ -306,27 +306,28 @@ function doUninstall() {
...
     
     [ -z "${project_path}" ] && doFail "Config is missing required parameter 'project.path'"
-    [ -z "${project_workspace}" ] && local project_workspace=main
+    [ -z "${project_workspace}" ] && project_workspace=main
     [ -z "${project_git_url}" ] && doFail "Config is missing required parameter 'project.git.url'"
-    [ -z "${project_eclipse}" ] && local project_eclipse=import
+    [ -z "${project_eclipse}" ] && project_eclipse=import
     
-    echo "Project (Path: ${project_path}, Url: ${project_git_url}, Branch: ${project_git_branch}, Eclipse: ${project_eclipse}, Workspace: ${project_workspace}, Workingsets: ${project_workingsets})"
-    if [ "${project_workspace}" == "${WORKSPACE}" ]
```
Comment (maihacke) [inline scripts/src/main/resources/scripts/command/ide]: why did you remove the if-statement?  now all projects will be imported into all workspaces...

Comment (hohwille) [reply scripts/src/main/resources/scripts/command/ide]: No each project is configured into the workspace that is configured.
Otherwise only the projects for the main workspace where perocessed during setup and all others had been ignored.

Comment (hohwille) [reply scripts/src/main/resources/scripts/command/ide]: OK. We now have clarified this in a call. Thanks as this revealed gaps on both of our sides so we can now fix everything properly.
